packages/backend/src/graphql/scalars/UInt8.test.ts

```diff
@@ -0,0 +1,154 @@
+import {
+  GraphQLObjectType,
+  GraphQLNonNull,
+  GraphQLInputObjectType
+} from 'graphql/type/definition'
+import { GraphQLSchema, graphql } from 'graphql'
+import { GraphQLUInt8 } from './UInt8'
+
+describe('UInt8', () => {
+  const Query = new GraphQLObjectType({
+    name: 'Query',
+    fields: {
+      mirror: {
+        type: new GraphQLNonNull(GraphQLUInt8),
+        args: {
+          num: { type: new GraphQLNonNull(GraphQLUInt8) }
+        },
+        resolve: (root, args) => args.num
+      },
+      typeBoolErr: {
+        type: new GraphQLNonNull(GraphQLUInt8),
+        resolve: () => true
+      },
+      typeFloatErr: {
+        type: new GraphQLNonNull(GraphQLUInt8),
+        resolve: () => 3.14
+      }
+    }
+  })
+
+  const Mutation = new GraphQLObjectType({
+    name: 'Mutation',
+    fields: {
+      mirror: {
+        type: new GraphQLNonNull(
+          new GraphQLObjectType({
+            name: 'MirrorPayload',
+            fields: {
+              result: { type: new GraphQLNonNull(GraphQLUInt8) }
+            }
+          })
+        ),
+        args: {
+          input: {
+            type: new GraphQLNonNull(
+              new GraphQLInputObjectType({
+                name: 'MirrorInput',
+                fields: {
+                  num: { type: new GraphQLNonNull(GraphQLUInt8) }
+                }
+              })
+            )
+          }
+        },
+        resolve: (root, args) => ({ result: args.input.num })
+      }
+    }
+  })
+
+  const schema = new GraphQLSchema({
+    query: Query,
+    mutation: Mutation
+  })
+
+  const validQuery = `{
+        a: mirror(num: 0)
+        b: mirror(num: 1)
+        c: mirror(num: 11)
+        d: mirror(num: 111)
+        e: mirror(num: 255)
+      }`
+
+  const floatQuery = `{
+        k: typeFloatErr
+      }`
+
+  const boolQuery = `{
+        k: typeBoolErr
+      }`
+
+  const validMutation = `mutation test(
+        $input1: MirrorInput!,
+        $input2: MirrorInput!,
+        $input4: MirrorInput!
+      ) {
+        a: mirror(input: $input1) { result }
+        b: mirror(input: $input2) { result }
+        d: mirror(input: $input4) { result }
+      }`
+
+  const validVariables = {
+    input1: { num: 5 },
+    input2: { num: '' },
+    input4: { num: '1' }
+  }
```

Comment:
Can these be moved inside their corresponding tests?
(Are these tests based on an example you found?)